Candidate: CVE-2019-18874
PublicDate: 2019-11-12 02:15:00 UTC
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-18874
 https://github.com/giampaolo/psutil/pull/1616
 https://github.com/giampaolo/psutil/pull/1624
Description:
 psutil (aka python-psutil) through 5.6.5 can have a double free. This
 occurs because of refcount mishandling within a while or for loop that
 converts system data into a Python object.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
 http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=944605
Priority: medium
Discovered-by: Riccardo Schirone
Assigned-to: mdeslaur

Patches_python-psutil:
 upstream: https://github.com/giampaolo/psutil/commit/7d512c8e4442a896d56505be3e78f1156f443465
 upstream: https://github.com/giampaolo/psutil/commit/3a9bccfd2c6d2e6538298cd3892058b1204056e0
upstream_python-psutil: needs-triage
precise/esm_python-psutil: DNE
trusty_python-psutil: ignored (out of standard support)
trusty/esm_python-psutil: DNE
xenial_python-psutil: needed
bionic_python-psutil: needed
disco_python-psutil: needed
eoan_python-psutil: needed
devel_python-psutil: needed
